README.md

```diff
@@ -108,7 +108,7 @@ saveCityZipCode(address.match(cityZipCodeRegex)[1], address.match(cityZipCodeReg
 ```javascript
 const address = 'One Infinite Loop, Cupertino 95014';
 const cityZipCodeRegex = /^[^,\\]+[,\\\s]+(.+?)\s*(\d{5})?$/;
-const [, city, zipCode] = address.match(cityZipCodeRegex) || [];
+const [city, zipCode] = address.match(cityZipCodeRegex) || [];
```

Comment:
Have you tried this?
The first element will be the _entire matched string_.
The comma is required to be there.

Reply:
Not I have not tried. My mistake. i was just thinking it was a typo. Sorry about that

Reply:
@jankoboz The entire string is matched because of the capturing groups, "( )" in the regex  :)